IBPSA/Electrical/Data/PV/SingleDiodeData.mo

```diff
@@ -0,0 +1,58 @@
+within IBPSA.Electrical.Data.PV;
+record SingleDiodeData
+  "Basic record of a PV module with single diode base model"
+  extends Modelica.Icons.Record;
+  extends IBPSA.Electrical.Data.PV.Generic;
+
+  parameter Modelica.Units.SI.Efficiency eta0
+    "Efficiency under standard conditions. If not found in data sheet, use eta_0 = ((V_mp0*I_mp0)/(1000*A_cel*n_ser))"
+    annotation (Dialog(group="General"));
+
+  parameter Modelica.Units.SI.ElectricCurrent IMP0
+    "MPP current under standard conditions"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.Power PMP0
+    "MPP power of one PV module under standard conditions"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Real TCoeISC(unit="A/K")
+    "Temperature coefficient for short circuit current, >0. If not found in data sheet, use TCoeff_Isc=alpha_Isc*I_sc0 and type in alpha_Isc manually"
+    annotation(Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Real TCoeVOC(unit="V/K")
+    "Temperature coefficient for open circuit voltage, <0. If not found in data sheet, use TCoeff_Voc=beta_Voc*V_oc0 and type in beta_Voc manually"
+    annotation(Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.LinearTemperatureCoefficient alphaISC=TCoeISC/
+      ISC0 "Normalized temperature coefficient for short circuit current, >0"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.LinearTemperatureCoefficient betaVOC=TCoeVOC/VOC0
+    "Normalized temperature coefficient for open circuit voltage, <0"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.LinearTemperatureCoefficient gammaPMP
+    "Normalized temperature coefficient for power at MPP"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.Voltage VOC0
+    "Open circuit voltage under standard conditions"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.ElectricCurrent ISC0
+    "Short circuit current under standard conditions"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Modelica.Units.SI.Voltage VMP0
+    "MPP voltage under standard conditions"
+    annotation (Dialog(group="Cell specific: Electrical characteristics"));
+  parameter Real C
```

Comment:
Shouldn't this have a unit?

Reply:
it has no unit. But I added the (unit="1") annotation